src/EnergyPlus/MixedAir.cc

```diff
@@ -4328,6 +4347,16 @@ namespace MixedAir {
 
 		}
 
+		if ( OAController( OAControllerNum ).CoolCoilFreezeCheck ) {
+			MaxOAFracBySetPoint = min( max( MaxOAFracBySetPoint, 0.0 ), 1.0 );
+			OAController( OAControllerNum ).MaxOAFracBySetPoint = MaxOAFracBySetPoint;
```

Comment:
I don't see l_ as a prefix anywhere else in the code.  @Myoldmopar what's our preferred style for this?